### PR TITLE
add minVAF for filtering mutations and add genomic tmb

### PR DESCRIPTION
**Pull request by xuemeiluo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,9 +6,11 @@ Tumour Mutation Burden scores are calculated from somatic variant calls coming o
 
 Accuracy of the TMB metric is dependent on having sufficient depth to ensure coverage over this region. We recommend at a mininum 80X on the tumour and 30X on the matched normal.
 
-TMB is the number of PASS somatic mutations per megabase within the target region.  The full exome target region should be used for WGS samples. The mutations are restricted to protein altering mutations of the following vep classes:
+The TMB is the number of PASS somatic mutations with the VAF > 10% per megabase within the target region.  The exome mutations are restricted to protein altering mutations of the following vep classes:
 
-Missense_Mutation,In_Frame_Ins,In_Frame_Del,Frame_Shift_Ins,Frame_Shift_Del,Splice_Site,Translation_Start_Site,Nonsense_Mutation,Nonstop_Mutation
+Missense_Mutation,In_Frame_Ins,In_Frame_Del,Frame_Shift_Ins,Frame_Shift_Del,Splice_Site,Translation_Start_Site,Nonsense_Mutation,Nonstop_Mutation.
+
+The Genome mutations are all mutations within the genome region.
```
Comment (bibilujan) [inline README.md:15]: I suggest to add "per megabase" to the second description:
The Genome mutations are all mutations per megabase within the genome region.

File: README.md [hunk 2]
```diff
@@ -55,32 +60,48 @@ Output | Type | Description
 
 
 ## Commands
- This section lists command(s) run by tmbAnalysis workflow
+ This section lists command(s) run by the tmbAnalysis workflow
  
- * Running tmbAnalysis
+ * Running WORKFLOW
```
Comment (bibilujan) [inline README.md]: This and the line after need updating, can be kept similar to the previous version.

File: tmbAnalysis.wdl [hunk 3]
```diff
@@ -27,7 +27,7 @@ workflow tmbAnalysis {
...
+    description: "Tumour Mutation Burden (TMB) Workflow\n\nTumour Mutation Burden scores are calculated from somatic variant calls coming out of the mutect2 workflow, after annotation with variant effect predictor (vep).  This requires a tumour sample with a matched normal.\n\nAccuracy of the TMB metric is dependent on having sufficient depth to ensure coverage over this region. We recommend at a mininum 80X on the tumour and 30X on the matched normal.\n\nThe TMB is the number of PASS somatic mutations with the VAF > 10% per megabase within the target region.  The exome mutations are restricted to protein altering mutations of the following vep classes:\n\nMissense_Mutation,In_Frame_Ins,In_Frame_Del,Frame_Shift_Ins,Frame_Shift_Del,Splice_Site,Translation_Start_Site,Nonsense_Mutation,Nonstop_Mutation.\n\nThe Genome mutations are all mutations within the genome region."
```
Comment (bibilujan) [inline tmbAnalysis.wdl]: add per megabase as previous comment in README

**Review by alexjfortuna — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (xuemeiluo) [reply README.md]: updated

File: README.md (hunk 1 above)
Comment (xuemeiluo) [reply README.md:15]: updated

File: tmbAnalysis.wdl (hunk 3 above)
Comment (xuemeiluo) [reply tmbAnalysis.wdl]: updated

**Review by bibilujan — APPROVED** [review]

**Pull request merged**